packages/app/src/Feed/RelaysFeed.tsx

```diff
@@ -13,7 +13,7 @@ export default function useRelaysFeed(pubkey: HexKey) {
     return x;
   }, [pubkey]);
 
-  const relays = useSubscription(sub, { leaveOpen: false, cache: true });
+  const relays = useSubscription(sub, { leaveOpen: false, cache: false });
```

Comment:
This doesn't seem to do the trick. I'm still seeing wss://nostr-dev.wellorder.net on my profile page. I even tried opening my profile page in a different browser. Is there any way to bust the cache?

Here's the link to my profile on this preview branch https://note-overflow.snort-social.pages.dev/p/npub1vp8fdcyejd4pqjyrjk9sgz68vuhq7pyvnzk8j0ehlljvwgp8n6eqsrnpsw.